external-resolver/src/test/scala/org/genivi/sota/resolver/test/PackagesResourceSpec.scala

```diff
@@ -74,6 +75,39 @@ class PackagesResourcePropSpec extends ResourcePropSpec with PackageGenerators {
     }
   }
 
+  property("Posting to affected packages returns affected devices") {
+    forAll { (device: Device.Id, p: Package) =>
+      addPackage(p.namespace, p.id.name.get, p.id.version.get, p.description, p.vendor) ~> route ~> check {
+        status shouldBe StatusCodes.OK
+      }
+
+      installPackage(device, p) ~> route ~> check {
+        status shouldBe StatusCodes.OK
+      }
+
+      getAffected(Set(p.id)) ~> route ~> check {
+        status shouldBe StatusCodes.OK
+        responseAs[Map[Device.Id, Seq[PackageId]]] should contain(device -> Seq(p.id))
```

Comment:
As above, I think matching exactly might be better here, to minimize the chance of false test passes.